src/test/rustdoc/auxiliary/external-macro-src.rs

```diff
@@ -0,0 +1,15 @@
+// compile-flags:--remap-path-prefix={{src-base}}=/does-not-exist
```

Comment:
Nice!

Reply:
Surprised `{{src-base}}` exists, based on the normalization rules I would've expected it to be `$SRC_DIR`.